src/Raven.Client/Documents/Session/IAdvancedSessionOperations.Stream.cs

```diff
@@ -47,6 +47,14 @@ public partial interface IAdvancedSessionOperations
         ///     <para>Does NOT track the entities in the session, and will not includes changes there when SaveChanges() is called</para>
         /// </summary>
         /// <param name="query">Query to stream results for</param>
+        IEnumerator<StreamResult<T>> Stream<T>(IRawDocumentQuery<T> query);
```

Comment:
Maybe also add the overload with `out StreamQueryStatistics streamQueryStats` ?

Reply:
Yes